ajax/libs/hyperHtml/package.json

```diff
@@ -0,0 +1,33 @@
+{
+  "name": "hyperHtml",
+  "filename": "min.min.js",
+  "description": "A Fast & Light Virtual DOM Alternative",
+  "license": "ISC",
+  "author": "Andrea Giammarchi",
+  "repository": {
+    "type": "git",
+    "url": "https://github.com/WebReflection/hyperhtml.git"
+  },
+  "keywords": [
+    "dom",
+    "diff",
+    "performance",
+    "template",
+    "literals",
+    "lightweight",
+    "fast",
+    "react",
+    "virtual",
+    "lit-html",
+    "alternative"
+  ],
+  "npmName": "hyperhtml",
+  "npmFileMap": [
+    {
+      "basePath": "dist",
+      "files": [
+        "**/*"
+      ]
+    }
```

Comment:
Suppose the file for a browser is only `min.js`. It's under the root directory, so the pattern here would be
```js
   {
      "basePath": "",
      "files": [
        "min*.+(css|js|map)"
      ]
    }

```

@WebReflection Could you tell me if `hyperhtml.js` in the old versions is the unminified version of `min.js`? If the answer is yes, we hope to include it in our auto-update config too.

Reply:
@extend1994 from https://unpkg.com/hyperhtml/ , the following are the files that makes sense to have in a CDN:

  * `/esm/**`
  * `/index.js`
  * `/min.js`
  * `/no.js`
  * `/umd.js`

Did I answer your question?

Reply:
@WebReflection Yes :+1: Just still have one question: `esm` files seem unable to be used in a browser? Like the way `<script src="*.js">`. Not sure if my knowledge is correct :sweat:

Reply:
Welcome to the future of JavaScript modules ;-)

```html
<!doctype html>
<script type=module>
import {bind} from 'https://unpkg.com/hyperhtml/esm/index.js';

window.onload = () => {
  bind(document.body)`Hello World`;
};
</script>
```

TL;DR Yes, `/esm/**/*` folder is usable right away to import ESM (ECMAScript Modules)

Reply:
@WebReflection Thanks a lot for the explanation! Good to know it :smile: However, I can't be sure that cdnjs can host this kind of files. To my knowledge, cdnjs hasn't hosted them yet.
@PeterDaveHello Could you please help check this? Thanks!

Reply:
FWIW, bundlers make the usage of these files via CDN somehow useless, so that having the following might be enough for the time being:

  * /index.js
  * /min.js
  * /no.js
  * /umd.js

Reply:
@PeterDaveHello May I know your opinion about the files of ECMAScript modules which is mentioned above?